test/framework/src/test/java/org/elasticsearch/test/test/InternalTestClusterTests.java

```diff
@@ -272,7 +272,7 @@ public Settings nodeSettings(int nodeOrdinal) {
                         NodeEnvironment.MAX_LOCAL_STORAGE_NODES_SETTING.getKey(),
                         2 + (masterNodes ? InternalTestCluster.DEFAULT_HIGH_NUM_MASTER_NODES : 0) + maxNumDataNodes + numClientNodes)
                     .put(NetworkModule.TRANSPORT_TYPE_KEY, getTestTransportType())
-                    .put(USE_ZEN2.getKey(), false) // full cluster restarts not yet supported
+                    .putList(DISCOVERY_HOSTS_PROVIDER_SETTING.getKey(), "file")
```

Comment:
also set `.putList(SettingsBasedHostsProvider.DISCOVERY_ZEN_PING_UNICAST_HOSTS_SETTING.getKey()) // empty list disables a port scan for other nodes`?